fetch.bs

```diff
@@ -4156,6 +4156,33 @@ these steps:
    <li><p>Let <var>requestForServiceWorker</var> be a <a for=request>clone</a> of
    <var>request</var>.
 
+   <li>
+    <p> If <var>requestForServiceWorker</var>'s <a for=/>body</a> is non-null, then:
+
+    <ol>
+     <li> Let <var>transformAlgorithm</var> given <var>chunk</var> be these steps:
```

Comment:
My understanding is that we should have a `<p>` after the `<li>` but I will defer to @annevk's judgement.

Reply:
Added `<p>`